crates/bevy_scene/src/dynamic_scene.rs

```diff
@@ -37,6 +39,57 @@ impl DynamicScene {
         Self::from_world(&scene.world, type_registry)
     }
 
+    /// Create a Dynamic Scene with specific entities.
+    ///
+    /// The generic parameter is used as a `Query` filter.
+    ///
+    /// The created scene will include only the entities that match the query
+    /// filter provided. All components that impl `Reflect` will be included.
+    pub fn from_query_filter<F>(world: &mut World) -> Self
```

Comment:
I changed the name because I thought `DynamicScene::from_query_filter` sounded nice and seems to fit with the style Bevy uses.

Reply:
I like this name!